Makefile

```diff
@@ -49,13 +49,11 @@ deploy-gh-pages:
 	rm -rf ${FRONTEND_TMP}
 	git worktree prune
 	GATSBY_BASE_URL=https://img.shields.io \
-		NEXT_ASSET_PREFIX=https://shields.io \
 		npm run build
 	git worktree add -B gh-pages ${FRONTEND_TMP}
 	git -C ${FRONTEND_TMP} ls-files | xargs git -C ${FRONTEND_TMP} rm
 	git -C ${FRONTEND_TMP} commit --no-verify -m '[DEPLOY] Completely clean the index'
-	cp -r build/* ${FRONTEND_TMP}
-	cp favicon.png ${FRONTEND_TMP}
```

Comment:
Just wanted to double check this removal was intentional

Reply:
Yea, the favicon is now embedded as a data URL.

Reply:
… which we should totally do with the badge previews 😁